lib/python/vdm/server/HTTPListener.py

```diff
@@ -27,14 +27,22 @@
 OTHER DEALINGS IN THE SOFTWARE.
 """
 
-from flask import Flask, render_template, jsonify, abort, make_response, request
+from flask import Flask, render_template, jsonify, abort, make_response, request, Response
 from flask.views import MethodView
-from Validation import ServerInputs, DatabaseInputs, JsonInputs, UserInputs
+from Validation import ServerInputs, DatabaseInputs, JsonInputs, UserInputs, ConfigValidation
 import socket
 import os
 import json
+from xml.etree.ElementTree import Element, SubElement, Comment, tostring, parse, XML
+import requests
+from flask import Flask
+from flask.ext.cors import CORS
+from collections import defaultdict
+import ast
+import os.path
 
 APP = Flask(__name__, template_folder="../templates", static_folder="../static")
```

Comment:
can flask be configured to bind to an interface and advertise its host name? if so they should be command line options